### PR TITLE
docs: update observability demo and cross links

### PR DESCRIPTION
**Pull request by zr-msft**

### CODE REVIEW

File: content/docs/demos/prometheus_grafana.md [hunk 1]
```diff
@@ -1,49 +1,61 @@
...
+## Deploy an example Prometheus instance
+
+Use `helm` to deploy a Prometheus instance to your cluster.
 
-## Deploying a Prometheus Instance
-The easiest way to deploy a Prometheus instance is through Helm:
 ```
 helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
 helm install stable prometheus-community/prometheus
```
Comment (jaellio) [inline content/docs/demos/prometheus_grafana.md:21]: Should we call out that we are using the default namespace?

File: content/docs/demos/prometheus_grafana.md [hunk 2]
```diff
@@ -1,49 +1,61 @@
...
-In the next step, we need to configure the targets to scrape. OSM uses additional re/labeling and endpoint configuration, we we strongly recommend replacing Prometheus configuration or otherwise the OSM Grafana dashboard might show incomplete data.
+## Configure Prometheus for OSM
 
-```
-kubectl get configmap
-NAME                             DATA   AGE
-stable-prometheus-alertmanager   1      22s
-stable-prometheus-server         5      22s
-```
+Prometheus needs to be configured to scape the OSM endpoints and properly handle OSM's labeling, relabelling, and endpoint configuration. This configuration also helps Grafana dashboard, which is configured in a later step, properly display the data scraped from OSM.
```
Comment (jaellio) [inline content/docs/demos/prometheus_grafana.md:37]:
```suggestion
Prometheus needs to be configured to scape the OSM endpoints and properly handle OSM's labeling, relabelling, and endpoint configuration. This configuration also helps the OSM Grafana dashboards, which are configured in a later step, properly display the data scraped from OSM.
```

File: content/docs/getting_started/observability.md [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
 title: "Observability"
-description: "Use OSM's observability integrations to inspect the traffic between the bookstore applications"
+description: "Use OSM's observability integrations with Prometheus and Grafana to inspect the traffic between the bookstore applications"
 type: docs
 weight: 5
 ---
 
 # Observability
 
+The following article shows you how to install OSM with automatic provisioning of Prometheus and Grafana stack for observability and monitoring. For an example using a bring your own (BYO) Prometheus and Grafana stack on your cluster with OSM, see the [Integrate OSM with Prometheus and Grafana](https://docs.openservicemesh.io/docs/demos/prometheus_grafana/) demo.
```
Comment (jaellio) [inline content/docs/getting_started/observability.md:10]:
```suggestion
The following article shows you how to install OSM with automatic provisioning of the Prometheus and Grafana stack for observability and monitoring. For an example using a bring your own (BYO) Prometheus and Grafana stack on your cluster with OSM, see the [Integrate OSM with Prometheus and Grafana](https://docs.openservicemesh.io/docs/demos/prometheus_grafana/) demo.
```

File: content/docs/demos/prometheus_grafana.md (hunk 1 above)
Comment (zr-msft) [reply content/docs/demos/prometheus_grafana.md:21]: fixed

File: content/docs/demos/prometheus_grafana.md (hunk 2 above)
Comment (zr-msft) [reply content/docs/demos/prometheus_grafana.md]: fixed

File: content/docs/getting_started/observability.md (hunk 3 above)
Comment (zr-msft) [reply content/docs/getting_started/observability.md]: fixed

**Comment by zr-msft** [comment]
thank you @jaellio  for the feedback! I've incorporated your changes. Let me know what you think

File: content/docs/demos/prometheus_grafana.md [hunk 4]
```diff
@@ -1,49 +1,61 @@
 ---
-title: "Prometheus & Grafana"
-description: "Describes basic steps to get up and running a Prometheus and Grafana stack with OSM specific configuration and Dashboards"
+title: "Integrate OSM with Prometheus and Grafana"
+description: "Describes how to set up an OSM-specific configuration and dashboards with your own Prometheus and Grafana stack"
 type: docs
 weight: 25
 ---
```
Comment (jaellio) [inline content/docs/demos/prometheus_grafana.md:7]: I think all other demos have a list of prerequisites at the beginning of the article. Could we add this here?

File: content/docs/demos/prometheus_grafana.md [hunk 5]
```diff
@@ -1,49 +1,61 @@
...
-title: "Prometheus & Grafana"
-description: "Describes basic steps to get up and running a Prometheus and Grafana stack with OSM specific configuration and Dashboards"
+title: "Integrate OSM with Prometheus and Grafana"
```
Comment (jaellio) [inline content/docs/demos/prometheus_grafana.md:2]: The other OSM demos have a slightly different format. For example, all demos begin like this:
image

And the `Integrate OSM with Prometheus and Grafana` demo is formatted like this:
image

I am not sure what the best formatting option is and this doesn't need to be addressed in this PR, but I think a consistent layout is helpful.

File: content/docs/demos/prometheus_grafana.md (hunk 4 above)
Comment (zr-msft) [reply content/docs/demos/prometheus_grafana.md:7]: fixed

File: content/docs/demos/prometheus_grafana.md (hunk 5 above)
Comment (zr-msft) [reply content/docs/demos/prometheus_grafana.md:2]: agreed. let me know what you think of the formatting for this demo and i can update the others as well

Comment (jaellio) [reply content/docs/demos/prometheus_grafana.md:2]: I like including the title and using the subtitle font for the description as long as it is brief.

File: content/docs/demos/prometheus_grafana.md [hunk 6]
```diff
@@ -1,49 +1,69 @@
...
 
-## Overview
-The following doc walks through the process of creating a simple Prometheus and Grafana stack to enable observability and monitoring of OSM's service mesh.
+The following article shows you how to create an example bring your own (BYO) Prometheus and Grafana stack on your cluster and configure that stack for observability and monitoring of OSM. For an example using an automatic provisioning of a Prometheus and Grafana stack with OSM, see the [Observability](https://docs.openservicemesh.io/docs/getting_started/observability/) getting started guide.
 
-This setup is not meant for production, for production-grade deployments one should refer to [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md).
+The configuration created in this article should not be used in production environments. For production-grade deployments, see [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/user-guides/getting-started.md) and [Deploy Grafana in Kubernetes](https://grafana.com/docs/grafana/latest/installation/kubernetes/).
```
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/docs/demos/prometheus_grafana.md]: changed to important for higher visibility. I didn't realize you had notes/callouts so thank you for seeing this

File: content/docs/getting_started/observability.md (hunk 3 above)
Comment (zr-msft) [reply content/docs/getting_started/observability.md]: fixed

File: content/docs/demos/prometheus_grafana.md (hunk 4 above)
Comment (zr-msft) [reply content/docs/demos/prometheus_grafana.md:7]: fixed

**Comment by jaellio** [comment]
LGTM! Could you please add a comment to the description saying you ran through the updated demo and verified it works?

**Review by jaellio — APPROVED** [review]

**Pull request merged**